CMakeLists.txt

```diff
@@ -0,0 +1,32 @@
+cmake_minimum_required(VERSION 3.23.1 FATAL_ERROR)
```

Comment:
[nvptxcompiler requires at least 3.25](https://cmake.org/cmake/help/latest/module/FindCUDAToolkit.html#nvptx-compiler). You may want to just require 3.26.4 since that's what RAPIDS is doing now.

Reply:
```suggestion
cmake_minimum_required(VERSION 3.26.4 FATAL_ERROR)
```